ElectricEngine.hpp

```diff
@@ -4,8 +4,14 @@ class ElectricEngine
 {
 public:
     ElectricEngine(int power, int batteryCapacity);
+    ~ElectricEngine();
+
+    int getGear();
+    bool isGearValid(int& gear);
 
     int power_;             // in HP 
     int batteryCapacity_;   // in Ah
+    int gears_;
```

Comment:
koncepcja była taka, że silnik elektryczny w ogóle nie ma biegów, ale nie ma problemu jeśli chceie to dodać. Ale to zbędny kod z mojego punktu widzenia.